src/Air.zig

```diff
@@ -553,6 +553,9 @@ pub const Inst = struct {
         /// Constructs a vector by selecting elements from `a` and `b` based on `mask`.
         /// Uses the `ty_pl` field with payload `Shuffle`.
         shuffle,
+        /// Constructs a vector element-wise from `a` or `b` based on `pred`.
+        /// Uses the `ty_pl` field with payload `Select`.
```

Comment:
@andrewrk should this be a `pl_op` instead? 🤔 The type is not used for the LLVM backend, but I don't know if it might be needed for the other backends.